scanner/go.mod

```diff
@@ -1,11 +1,12 @@
-module github.com/stackrox/scanner/v4
+module github.com/stackrox/stackrox/scanner/v4
```

Comment:
I think a few go files will need to be updated for this. Wasn't caught by CI because we don't build this module in CI yet

Reply:
Yeah, we will fix it as it comes to it.